website/cue/reference/releases/0.29.0.cue

```diff
@@ -0,0 +1,594 @@
+				events.
+				"""
+			contributors: ["nabokihms"]
+			pr_numbers: [16769]
+		},
+		{
+			type: "fix"
+			scopes: ["statsd sink"]
+			description: """
+				The `statsd` sink now supports encoding aggregate histograms. Not supported, yet, is
+				encoding sketches as histograms.
+				"""
+			contributors: ["pedro-stanaka"]
+			pr_numbers: [14762]
+		},
+		{
+			type: "fix"
+			scopes: ["codecs"]
+			description: """
+				Encoding of timestamps by the `gelf` codec was fixed to correctly encode as an
+				integer rather than an an RFC3339 string.
+				"""
+			contributors: ["scMarkus"]
+			pr_numbers: [16749]
+		},
+		{
+			type: "fix"
+			scopes: ["releasing"]
+			description: """
+				The distributed SystemD unit file now has `Restart` set to `Always` to match
+				expected user behavior when Vector exits.
+				"""
+			contributors: ["KannarFr"]
+			pr_numbers: [16822]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability"]
+			description: """
+				Vector now logs signals that it receives, and handles, at the INFO level. Unhandled
+				signals are ignored.
+				"""
+			contributors: ["zamazan4ik"]
+			pr_numbers: [16835]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl: stdlib"]
+			description: """
+				An `hmac` function was added to VRL to calculate
+				[HMACs](https://en.wikipedia.org/wiki/HMAC).
+				"""
+			contributors: ["sbalmos"]
+			pr_numbers: [15371]
+		},
+		{
+			type: "fix"
+			scopes: ["core"]
+			description: """
+				Comparisons of float fields in Vector was fixed to completely compare the value.
+				Previously it was only comparing the integer portion. This mostly came up in VRL
+				when doing field comparisons like `.foo == .bar` where `.foo` and `.bar` were float
+				values.
+				"""
+			pr_numbers: [16926]
+		},
+		{
+			type: "fix"
+			scopes: ["prometheus_scrape source"]
+			description: """
+				The `prometheus_scrape` source now handles larger counts for summary metrics (up to
+				uint64 instead of uint32).
+				"""
+			contributors: ["uralmetal"]
+			pr_numbers: [16946, 16508]
+		},
+		{
+			type: "feat"
+			scopes: ["vrl: stdlib"]
+			description: """
+				A few functions to generate random values were added to VRL:
+
+				- `random_bool`: generate a random boolean
+				- `random_int`: generate a random integer
+				- `random_float`: generate a random float
+				"""
+			pr_numbers: [16768]
+		},
+		{
+			type: "feat"
+			scopes: ["aws_ecs_metrics source"]
+			description: """
+				The `aws_ecs_metrics` source now collects `precpu_online_cpus` stats.
+				"""
+			contributors: ["asingh072318"]
+			pr_numbers: [16768]
+		},
+		{
+			type: "feat"
+			scopes: ["file source"]
+			description: """
+				The `file` source now handles acknowledgements when Vector is shutting down and
+				sinks are flushing events. Previously the `file` source would ignore all
+				acknowledgements once Vector starts to shut down.
+				"""
+			contributors: ["jches"]
+			pr_numbers: [16928]
+		},
+		{
+			type: "fix"
```

Comment:
Also arguably a feat.

Reply:
This one I'd argue is a bug since I'd have expected retry back-offs already.

Reply:
Yeah, feels like it'd generally be an expected feature - we never said it would back-off though 🤣